ts-client/src/vault/index.ts

```diff
@@ -0,0 +1,82 @@
+import { AnchorProvider, Program } from '@project-serum/anchor';
+import { StaticTokenListResolutionStrategy } from '@solana/spl-token-registry';
+import { Connection, PublicKey } from '@solana/web3.js';
+import fetch from 'cross-fetch';
+import { IDL, Vault as VaultIdl } from "./idl";
+
+import { KEEPER_URL, PROGRAM_ID } from './constants';
+import { VaultAPY, VaultInfo } from './types';
+import { VaultImpl } from './vaultImpl';
+
+const tokenResolver = new StaticTokenListResolutionStrategy()
+    .resolve();
+
+type AllVaults = Array<{
+    data: VaultInfo & { apyState: VaultAPY },
+    vault: VaultImpl
+}>
```

Comment:
Can u move this to `types` directory and export it?  Need it on fe side to define the type